bases/br_bd_diretorios_brasil/escolas/table_config.yaml

```
@@ -0,0 +1,137 @@
+source_bucket_name: basedosdados-dev
+project_id_staging: basedosdados-dev
+project_id_prod: basedosdados-dev
+table_id: escolas # AUTO GENERATED
+dataset_id: br_bd_diretorios_brasil # AUTO GENERATED
+
+url_ckan: https://basedosdados.org/dataset/br-bd-diretorios-brasil # AUTO GENERATED
+url_github: https://github.com/basedosdados/mais/tree/master/bases/br_bd_diretorios_brasil # AUTO GENERATED
+
+version: <vA.B> # REQUIRED
+
+last_updated: 2021-02-25 # AUTO GENERATED
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.
+description: |  # REQUIRED
+    A tabela contém uma compatibilização de códigos nacionais e estaduais e nomes de escolas.
+
+# Quem está completando esse arquivo config?
+published_by:
+    name: Matheus Valentim de Barros  # REQUIRED
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_bd_diretorios_brasil/code # REQUIRED
+    website: https://github.com/mavalentim
+    email: mavalentim.b@gmail.com
+
+# Qual organização/departamento/pessoa tratou os dados?
+# As vezes há um ponto intermediário entre os dados originais e subir na Base dos Dados.
+# Se essa pessoa é você, preencha abaixo com suas informações.
+treated_by:
+    name: <nome>
+    code_url: <onde encontrar código de tratamento>
+    website: <onde encontrar os dados tratados>
+    email: <email>
+
+# Se houve passos de tratamento, limpeza e manipulação de dados, descreva-os aqui.
+treatment_description: | 
+    Juntei dados de diferentes bases para criar um diretório de compatibilização. Em maior detalhe, peguei dados de identificação de todas as escolas 
+    da BigQuery da Base dos Dados e fiz um left_join com os dados tratados provenientes da Secretaria de Educação de São Paulo. O join foi feito pela
+    id_escola, variável que identifica um código para cada escola a nível nacional.
+
+# Com qual frequência a base é atualizada?
+# Opções: hora | dia | semana | mes | 1 ano | 2 anos | 5 anos | 10 anos | unico | recorrente
+data_update_frequency: unico # REQUIRED 
```

aqui seria anual nao? o censo escolar disponibiliza anualmente a lista de escolas atualizadas
```suggestion
data_update_frequency: 1 ano # REQUIRED 
```

---

eu usei como base os outros diretórios dentro do dataset, vou alterar!